src/App.js

```diff
@@ -61,16 +65,47 @@ function App() {
     <div className="App">
       <header className="App-header">
         <h1>project+dendrology</h1>
+        <NavBar />
       </header>
       <main>
-        <QuestionContainer questions={randomizeQuesitons} position={position} />
-        <Form submitValue={checkAnswer} />
-        <button disabled={disableButton ? false : true} onClick={nextQuestion}>Next</button>
-        {correctAnswer ? <CorrectAnswer /> : ''}
-        {incorrectAnswer ? <IncorrectAnswer /> : ''}
-        {incrementCorrectAnswer}
-        {decrementAnswer}
-        {/* < ReviewQuestionsContainer reviewQuestions={reviewQuestions} position={position}/> */}
+      <Route exact path='/' render={() => 
+          <WelcomePage 
+            questions={randomizeQuesitons} 
+            position={position} 
+          />
+        } />
+        <Route exact path='/test' render={() => 
+          <QuestionContainer 
+            questions={randomizeQuesitons} 
+            position={position} 
+          />
+        } />
+        <Route exact path='/test' render={() => 
+          <Form 
+            submitValue={checkAnswer} 
+          />
+        } /> 
+        <Route exact path='/test' render={() => 
+          <button disabled={disableButton ? false : true} onClick={nextQuestion}>Next</button>
+        } />  
+          {correctAnswer ? <Route exact path='/test' render={() => 
+          <CorrectAnswer /> }
```

Comment:
Try wrapping all these elements in a div to remove all the duplicate "test" routes